diff --git a/node_modules/@op-engineering/op-sqlite/android/src/main/java/com/op/sqlite/OPSQLiteModule.kt b/node_modules/@op-engineering/op-sqlite/android/src/main/java/com/op/sqlite/OPSQLiteModule.kt
index 8d341b7..4f4d591 100644
--- a/node_modules/@op-engineering/op-sqlite/android/src/main/java/com/op/sqlite/OPSQLiteModule.kt
+++ b/node_modules/@op-engineering/op-sqlite/android/src/main/java/com/op/sqlite/OPSQLiteModule.kt
@@ -39,18 +39,12 @@ internal class OPSQLiteModule(context: ReactApplicationContext?) : ReactContextB
     }
 
     @ReactMethod(isBlockingSynchronousMethod = true)
-    fun install(): Boolean {
-        return try {
-            OPSQLiteBridge.instance.install(reactApplicationContext)
-            true
-        } catch (exception: Exception) {
-            Log.e(NAME, "Install exception: $exception")
-            false
-        }
+    fun install() {
+        OPSQLiteBridge.instance.install(reactApplicationContext)
     }
 
     @ReactMethod(isBlockingSynchronousMethod = true)
-    fun getDylibPath(bundleId: String, name: String): Boolean {
+    fun getDylibPath(bundleId: String, name: String) {
         throw Exception("Do not call getDylibPath on Android")
     }
 
diff --git a/node_modules/@op-engineering/op-sqlite/lib/module/functions.js b/node_modules/@op-engineering/op-sqlite/lib/module/functions.js
index 05fc2b0..68e4a96 100644
--- a/node_modules/@op-engineering/op-sqlite/lib/module/functions.js
+++ b/node_modules/@op-engineering/op-sqlite/lib/module/functions.js
@@ -7,10 +7,7 @@ if (global.__OPSQLiteProxy == null) {
   }
 
   // Call the synchronous blocking install() function
-  const installed = NativeModules.OPSQLite.install();
-  if (!installed) {
-    throw new Error(`Failed to install op-sqlite: The native OPSQLite Module could not be installed! Looks like something went wrong when installing JSI bindings, check the native logs for more info`);
-  }
+  NativeModules.OPSQLite.install();
 
   // Check again if the constructor now exists. If not, throw an error.
   if (global.__OPSQLiteProxy == null) {
